src/codegen/tests/expression_tests.rs

```diff
@@ -291,7 +291,60 @@ fn builtin_function_call_adr() {
     );
     // WHEN compiled
     // We expect the same behaviour as if REF was called, due to the assignee being a pointer
-    filtered_assert_snapshot!(result);
+    filtered_assert_snapshot!(result, @r#"
+    ; ModuleID = '<internal>'
+    source_filename = "<internal>"
+    target datalayout = "[filtered]"
+    target triple = "[filtered]"
+
+    %main = type { i32*, i32 }
+
+    @main_instance = global %main zeroinitializer
+
+    define void @main(%main* %0) {
+    entry:
+      %a = getelementptr inbounds %main, %main* %0, i32 0, i32 0
+      %b = getelementptr inbounds %main, %main* %0, i32 0, i32 1
+      store i32* %b, i32** %a, align 8
+      ret void
+    }
+    "#);
+}
+
+#[test]
+fn builtin_function_call_adr_with_named_argument() {
```

Comment:
Can you remove this test and move the `a := ADR(IN := b)` assignment into the `builtin_function_call_adr` function?